package.json

```diff
@@ -35,6 +35,7 @@
     "benderjs-promise": "^0.1.0",
     "benderjs-sinon": "^0.3.0",
     "chai": "^3.4.0",
+    "ckeditor5-dev-utils": "ckeditor/ckeditor5-dev-utils",
```

Comment:
Package is located under `devDependencies` and it feels like a natural way to use such dev-utils package. But this way `gulp-init` task will not put this package in development mode (it only takes `dependencies` from `package.json`). Is this intentional? This is utilities package but it is still developed by us, so this might be helpful. Maybe we need some option for `gulp-init` task to also scan `devDependencies`?